### DISCUSSION TITLE
Is there any way to disable Screen Capture/Screen Recording

### DISCUSSION THREAD

**Discussion by Nishank00** [halildurmus/win32#351] [Q&A] [answered]
I am trying to secure my application, Is there any way of disabling screen capture in win32 application. 

Thanks in advance

Example Image:
![image](https://user-images.githubusercontent.com/41135445/149748823-6cecc6a3-1fbc-4b0f-9533-f912f58bbba8.png)

**Comment by timsneath** [comment, marked as answer]
Yup -- the API in the accompanying article should work here:
https://yeasir007.blogspot.com/2019/10/how-to-prevent-disable-protect-screen-capture-of-windows-application.html

You should be able to do something like:

```dart
final hwnd = GetForegroundWindow();
final result = SetWindowDisplayAffinity(hwnd, WDA_MONITOR);
```

**Reply by Nishank00** [reply]
I was able to achieve the output using the above API, I have explained how to use it in the link below:

https://stackoverflow.com/questions/70739652/is-there-any-way-to-disable-screen-capture-screen-recording-flutter-win32-app
